artiq/frontend/artiq_coremgmt.py

```diff
@@ -132,9 +140,25 @@ def main():
             for key, filename in args.file:
                 with open(filename, "rb") as fi:
                     mgmt.config_write(key, fi.read())
+        if args.action == "write_channel_name":
+            ddb = DeviceDB(args.device_db).get_device_db()
+            for device, value in ddb.items():
+                try:
+                    channel = value["arguments"]["channel"]
+                    mgmt.config_write("channel "+str(channel), device.encode("utf-8"))
+                except:
```

Comment:
As I told you before, try/except blocks should:
1. encompass as few lines of code as possible
2. almost always filter by exception type

Right now if you have e.g. a typo in the call to ``mgmt.config_write`` then the code will silently do nothing. This is unacceptable.